src/SDKs/Batch/Management/changelog.md

```diff
@@ -1,5 +1,10 @@
 ## Microsoft.Azure.Management.Batch release notes
 
+### Changes in 6.0.0
+#### REST API version
+- This version targets REST API version 2017-09-01.
+- Batch ARM support certificate and pool operations.
```

Comment:
Rephrase to: "Adding support for Certificate and Pool operations"

Reply:
Fixed.